frontend/src/features/public-form/components/FormFields/SectionSidebar.tsx

```diff
@@ -20,7 +38,35 @@ export const SectionSidebar = (): JSX.Element => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [miniHeaderRef?.current?.clientHeight])
 
-  return (
+  return isMobile && isMobileSectionSidebarOpen && activeSectionId ? (
```

Comment:
Just checking: Why do we need to test for `activeSectionId`?

Reply:
I included it because the drawer should only appear if a form has sections. It's an extra check though - the [hamburger icon doesn't appear](https://github.com/opengovsg/FormSG/pull/4095/files#diff-ec12ea63e5d5583c3a961c60bb92052b4caa9da5181acc53f28c3eed9614af3eR124) if the form doesn't have sections, so technically there shouldn't be a case of the drawer appearing without the hamburger icon to trigger it. Do you think it makes more sense to remove this check in `SectionSidebar`?